tools/Makefile

```diff
@@ -287,11 +287,13 @@ install::
 
 # The bytecode disassembler
 
-DUMPOBJ=opnames.cmo dumpobj.cmo
+DUMPOBJ= \
+          ../compilerlibs/ocamlcommon.cma \
+          ../compilerlibs/ocamlbytecomp.cma \
+          \
```

Comment:
Are such blank lines really necessary?

Reply:
(In fact maybe you could pull this change---the one for `tools/Makefile` that is---into a separate GPR?)